addons/web/static/src/js/core/registry.js

```diff
@@ -68,7 +68,14 @@ var Registry = Class.extend({
      * @returns {Object}
      */
     entries: function () {
-        return Object.create(this.map);
+        var entries = {};
+        var keys = this.keys();
+        var key;
+        for (var i = 0; i < keys.length; i++) {
+            key = keys[i];
```

Comment:
`var key = keys[i];`
or even
`entries[keys[i]] = this.map[keys[i]];`

Reply:
Done